lib/core/toolrepos/autoupdate/ih_auto_update_repositories.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+repos=(engineering image-builder)
+
+for repo in "${repos[@]}" ; do
+    cd "$IH_HOME/$repo" || exit 1
+
+    # Saves our current and main/master branch of the repository
+    currentBranch=$(git branch --show-current)
+    mainBranch=$(git branch | tr " " "\n" | grep -E "(main|master)")
```

Comment:
`image-builder` and `engineering` both use `master` as the default branch.

For future reference, `git  branch -rl '*/HEAD' | rev | cut -d/ -f1 | rev` gets you the name of the default branch. I found that somewhere. It's worth figuring out what's actually going on there, it's very clever.